Library/Homebrew/attestation.rb

```diff
@@ -44,6 +44,12 @@ class InvalidAttestationError < RuntimeError; end
     # @api private
     class GhAuthNeeded < RuntimeError; end
 
+    # Raised if attestation verification cannot continue due to invalid
+    # credentials.
+    #
+    # @api private
+    class GhAuthInvalid < RuntimeError; end
```

Comment:
Maybe we can fold these two together? We don't seem to be using `GhAuthNeeded` now.

Reply:
`GhAuthNeeded` is still raised on line 124 when the credentials are not supplied at all. The messaging is slightly different between the two.

Reply:
Okay, makes sense. NB that this is probably a bug in `gh` itself so we may be able to remove this and treat it all as the same error in the near-ish future: https://github.com/cli/cli/issues/9338